### PR TITLE
[master]Pin webhook to v0.2.0-alpha1 in dockerfile

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

File: package/Dockerfile [hunk 1]
```diff
@@ -39,7 +39,7 @@ ENV KUSTOMIZE_VERSION v3.5.4
 # System charts minimal version
 ENV CATTLE_FLEET_MIN_VERSION=0.3.500
 ENV CATTLE_RANCHER_OPERATOR_MIN_VERSION=0.1.300
-ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=0.1.0-beta900
+ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=0.1.000
```
Comment (dramich) [inline package/Dockerfile:42]: I think master should be on https://github.com/rancher/webhook/releases/tag/v0.2.0-alpha01

Comment (StrongMonkey) [reply package/Dockerfile:42]: oh, good point

**Review by dramich — DISMISSED** [review]

**Review by cbron — COMMENTED** [review]
Whats going on with this one, just needs a rebase ?

**Comment by StrongMonkey** [comment]
@cbron It was failing at flaky tests. I will remove the test commit

**Review by cbron — APPROVED** [review]

**Pull request merged**